style.css/general.css

```diff
@@ -0,0 +1,33 @@
+html,
+body {
+  height: 100%;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  line-height: 29px;
+}
+
+.content {
+  flex: 1 0 auto;
+}
+
+.site-wrapper {
+  max-width: 1200px;
+  padding: 0 16px;
```

Comment:
You need the margin: 0 auto; as well. I asked to add the padding not replacing with the margin.

Reply:
Done.